internal/plugin/service/organization/organization_user_group_member.go

```diff
@@ -152,16 +152,16 @@ func (r *organizationUserGroupMembersResource) fillModel(
 		return nil
 	}
 
-	var member *aiven.OrganizationUserGroupMember
+	var member aiven.OrganizationUserGroupMember
 
 	for _, m := range list.Members {
 		if m.UserID == model.UserID.ValueString() {
-			member = &m
+			member = m
 			break
 		}
 	}
 
-	if member == nil {
+	if member.UserID == "" {
```

Comment:
this isn't clear why we are switching from the usage of the pointer here as well, could you please elaborate?